classes/form/CustomerForm.php

```diff
@@ -140,7 +140,8 @@ public function validate()
         }
 
         $passwordField = $this->getField('password');
-        if (Validate::isPasswd($passwordField->getValue()) === false) {
+        if (!empty($passwordField->getValue())
+            && Validate::isPasswd($passwordField->getValue()) === false) {
```

Comment:
```suggestion
        if ((!empty($passwordField->getValue()) || $this->passwordRequired)
            && Validate::isPasswd($passwordField->getValue()) === false) {
```

Reply:
Nice catch ! I did not know about this variable